library/Vanilla/Embeds/SoundCloudEmbed.php

```diff
@@ -65,7 +66,7 @@ public function renderData(array $data): string {
     /**
      * Parses the oembed repsonse html for permalink and other data.
```

Comment:
We should add a larger detailed comment here about the different types of soundcloud embeds we are trying to parse here and give an example of each type.